applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -82,9 +81,6 @@ def _ValidateSettings(self, settings):
             "linear_solver_settings": {
                 "solver_type": "amgcl"
             },
-            "formulation": {
-                "element_type": "incompressible"
-            },
```

Comment:
Why is this one removed?

Reply:
Because it was duplicated, check line 71